workspaces/adoption-insights/plugins/adoption-insights/src/utils/utils.ts

```diff
@@ -0,0 +1,186 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  format,
+  startOfToday,
+  startOfYear,
+  subDays,
+  isToday,
+  isYesterday,
+} from 'date-fns';
+
+export const getDateRange = (value: string) => {
```

Comment:
I feel we might be able put some of these into the common plugin once we have it.